enterprise/helpers.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Helper functions for enterprise app.
+"""
+
+from .models import EnterpriseCustomerBrandingConfiguration
+
+
+def get_branding_info(provider_id=None):
+    """
+    Return the EnterpriseCustomer branding information.
+    """
+    info = None
+    try:
+        info = EnterpriseCustomerBrandingConfiguration.objects.filter(
```

Comment:
I will prefer `get` instead of fetch--first(). As due to dependency this PR is in WIP state.